Documentation/Diagnostics/PH2132.md

```diff
@@ -0,0 +1,40 @@
+# PH2132: Remove commented code
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2132 |
+| Category  | [Documentation](../Documentation.md) |
+| Analyzer | [UnmanagedObjectsNeedDisposingAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/RemoveCommentedCodeAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+Unused code should not be commented. For historical reference, use a version control system instead.
+
+## How to solve
+
+Remove the comment containing unused source code.
+
```

Comment:
Consider giving away the secret that this analyzer works by looking for trailing semi-colons.

Reply:
No way! We might extend it later with { and }, as Sonar does